services/m3aggregator/config/aggregator.go

```diff
@@ -263,19 +263,20 @@ func (c *AggregatorConfiguration) NewAggregatorOptions(
 		opts = opts.SetMaxTimerBatchSizePerWrite(c.MaxTimerBatchSizePerWrite)
 	}
 
-	// Set default policies.
-	policies := make([]policy.Policy, len(c.DefaultPolicies))
+	// Set default storage policies.
+	// TODO(xichen): sort the storage policies.
+	policies := make([]policy.StoragePolicy, len(c.DefaultPolicies))
 	copy(policies, c.DefaultPolicies)
-	sort.Sort(policy.ByResolutionAscRetentionDesc(policies))
-	opts = opts.SetDefaultPolicies(policies)
+	// sort.Sort(policy.ByResolutionAscRetentionDesc(policies))
```

Comment:
still need this?

Reply:
Yeah I'll leave it as is, will clean it up when I address the TODOs.